git-cms-merge-topic

```diff
@@ -6,17 +6,20 @@ case `uname` in
 esac
 
 usage() {
-  $ECHO "git cms-merge-topic [options] <github-user>:<branch>"
-  $ECHO "git cms-merge-topic [options] <official-cmssw-branch>"
-  $ECHO "git cms-merge-topic [options] <pull-request-id>"
+  COMMAND_NAME=$1
+  $ECHO "git $COMMAND_NAME [options] <github-user>:<branch>"
+  $ECHO "git $COMMAND_NAME [options] <official-cmssw-branch>"
+  $ECHO "git $COMMAND_NAME [options] <pull-request-id>"
   $ECHO
   $ECHO "Options:"
   $ECHO "-d, --debug        \tenable debug output"
   $ECHO "    --https        \taccess GitHub over https (default)"
   $ECHO "    --ssh          \taccess GitHub over ssh"
-  $ECHO "    --no-commit    \tdo not do the final commit when merging"
-  $ECHO "-s, --strategy     \tspecify strategy when merging"
-  $ECHO "-X, --strategy-option     \tspecify strategy when merging"
+  if [ "$COMMAND_NAME" = "cms-merge-topic" ]; then
+    $ECHO "    --no-commit    \tdo not do the final commit when merging"
+    $ECHO "-s, --strategy     \tspecify strategy when merging"
+    $ECHO "-X, --strategy-option     \tspecify strategy when merging"
+  fi
```

Comment:
@kpedro88 your approach does not show `-n|--no-merge` among the options for `git-cms-merge-topic`, and shows a misleading usage information for `git-cms-merge-topic -n`